infra/resources/modules/commons/function_api.tf

```diff
@@ -27,21 +27,25 @@ locals {
 
     LEASES_COSMOSDB_CONTAINER_NAME = azurerm_cosmosdb_sql_container.leases.name
 
-    SUBSCRIPTION_HISTORY_CONSUMER                = "off"
-    SUBSCRIPTION_HISTORY_COSMOSDB_CONTAINER_NAME = azurerm_cosmosdb_sql_container.subscription_history.name
+    SUBSCRIPTION_HISTORY_CONSUMER                        = "off"
```

Comment:
Why off ?

Reply:
because this is the function that runs the API, not the consumers.

we deploy two functions, using the same artifact: the api function, inside the `function_api.tf` file, runs only the HTTP endpoints; the consumer function, inside the `function_consumptions.tf`, runs the consumers.

that's why the consumers are set to `off` in the API function.

Reply:
ok :)